lib/child_process.js

```diff
@@ -109,7 +97,6 @@ function normalizeExecArgs(command /*, options, callback*/) {
 exports.exec = function(command /*, options, callback*/) {
   var opts = normalizeExecArgs.apply(null, arguments);
   return exports.execFile(opts.file,
-                          opts.args,
```

Comment:
Here is what confuses me:

1. exec() calls execFile() without an args argument.

2. execFile() calls spawn() with an empty array as its args argument.

3. spawn() calls normalizeSpawnArguments() with that empty array as _its_ args argument.

4. normalizeSpawnArguments() returns that empty array in an options object.

5. The `opts.args` from that object is passed to child.spawn().

IOW, I get the impression that the `opts.args` from normalizeExecArgs() is lost.  I assume that's not the case but I don't understand why not.